packages/base/src/index.mjs

```diff
@@ -417,7 +417,20 @@ const rules = createConfig({
     ],
     'jsdoc/valid-types': 'error',
 
-    // 'promise/no-multiple-resolved': 'error',
+    /* promise plugin rules */
+    'promise/catch-or-return': [
+      'error',
+      {
+        allowFinally: true,
+      },
+    ],
```

Comment:
By default, `allowFinally` is set to `false`, meaning that this is not allowed unless returned.

```ts
someFunction().catch(() => { /* ... */ }).finally(() => { /* ... */ });
```

While this is allowed:

```ts
someFunction().catch(() => { /* ... */ });
```

or

```ts
return someFunction().catch(() => { /* ... */ }).finally(() => { /* ... */ });
```

After this change, the first example is allowed as well.

Reply:
That is a strange default. What does `finally` have to do with remembering to catch? Bizarre.